packages/form-config-antd/src/components/DraftFileList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -94,7 +94,7 @@ exports[`components/Antd/DraftFileList renders correctly: table 1`] = `
       },
     ]
   }
-  dataSource={
+  datasource={
```

Comment:
why?

Reply:
because in Table Component we have dataSource In lower Case